### PR TITLE
Collapsible Categories 2 Update

### PR DESCRIPTION
**Pull request by iMattPro**
@kasimi

### CODE REVIEW

**Comment by Galixte** [comment]
Thanks.

**Comment by bug-i** [comment]
I just tried your patch - this fix to conflicts with another extension (similar topics), where collapsing/expanding isnt working anymore.

**Comment by iMattPro** [comment]
Bump

File: styles/prosilver/template/mchat_body.html [hunk 1]
```diff
@@ -45,11 +45,14 @@ <h2>{{ lang('MCHAT_ARCHIVE_PAGE') }}</h2>
...
 				{% if MCHAT_IS_COLLAPSIBLE %}
 					<a href="{{ U_MCHAT_COLLAPSE_URL }}"
-						class="collapse-btn collapse-{{ S_MCHAT_HIDDEN ? 'show' : 'hide' }} mchat-collapse"
+						class="collapse-btn mchat-collapse"
 						data-hidden="{{ S_MCHAT_HIDDEN }}"
 						data-ajax="phpbb_collapse"
 						data-overlay="true"
-						title="{{ lang('MCHAT_COLLAPSE_TITLE') }}"></a>
+						data-title-alt="{{ lang('MCHAT_COLLAPSE_TITLE', ((not S_MCHAT_HIDDEN) * 1)) }}"
+						title="{{ lang('MCHAT_COLLAPSE_TITLE', (S_MCHAT_HIDDEN * 1)) }}">
```
Comment (kasimi) [inline styles/prosilver/template/mchat_body.html]: Please use `S_MCHAT_HIDDEN ? 1 : 0` instead of boolean multiplication.

**Review by kasimi — CHANGES_REQUESTED** [review]

**Comment by iMattPro** [comment]
Updated

**Pull request merged**